src/Sylius/Bundle/ApiBundle/Resources/config/services/state_providers.xml

```diff
@@ -16,6 +16,23 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
     <services>
+        <service id="sylius_api.state_provider.recursive_adjustments.order" class="Sylius\Bundle\ApiBundle\StateProvider\RecursiveAdjustmentsStateProvider">
```

Comment:
```suggestion
        <service id="sylius_api.state_provider.order_recursive_adjustments" class="Sylius\Bundle\ApiBundle\StateProvider\RecursiveAdjustmentsStateProvider">
```
🤔

Reply:
What is a reasoning behind adding the `order` part here?